test/avi/t_command_line_mode.clj

```diff
@@ -29,7 +29,7 @@
     (editor :after ":q") => unfinished?)
   (fact "`:q<Enter>` exits Avi"
     (editor :after ":q<Enter>") => finished?)
-  (fact "`:q!<Enter>` does not exit Avi"
+  (future-fact "`:q!<Enter>` does not exit Avi"
```

Comment:
In general, if you see a test like this, it's calling out that the behavior is intentional.  In this case, though, we were trying to come up with a different model for saving, but never succeeded.  So I would just change this to a

```clojure
(fact "`:q!<Enter>` exits Avi"
  ...
```